### ISSUE TITLE
Dockerized version

### ISSUE THREAD

**Issue by Jeroen-G** [avencera/rustywind#34]
Hi!

I'd just like you to know that I made a Dockerized version of Rustywind: https://github.com/Jeroen-G/rustywind-docker

This means that you can run it anywhere like this:

```
docker run --rm -v $PWD:/app jeroeng/rustywind:latest . --write
```

Thanks for the awesome tool!

**Comment by praveenperera** [comment]
That's great thanks! Would you mind adding the Dockerfile and the workflow to this repo as PR. 

That way we could keep the two versions up to date. And I will add it to the README.

**Comment by praveenperera** [comment]
I've created a Dockerfile in this repo: https://github.com/avencera/rustywind/blob/master/Dockerfile

It uses just alpine so the docker image is just 12MB vs 120MB for the node version

**Comment by Jeroen-G** [comment]
Nice finding, even though docker hub compresses it to 40mb the smaller the container the better!
I will PR the workflow tomorrow morning for you.

**Comment by Jeroen-G** [comment]
Only decision you need to make: Docker Hub or Github container registry?

**Comment by praveenperera** [comment]
Let’s do docker hub, less typing for people.

**Comment by praveenperera** [comment]
Released in v0.12.2

https://github.com/avencera/rustywind/releases/tag/v0.12.2

To GHCR: https://github.com/avencera/rustywind/pkgs/container/rustywind

And DockerHub: https://hub.docker.com/repository/docker/avencera/rustywind
